pandas/tests/frame/methods/test_reset_index.py

```diff
@@ -420,6 +424,37 @@ def test_reset_index_multiindex_columns(self):
         result = df2.rename_axis([("c", "ii")]).reset_index(col_level=1, col_fill="C")
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("flag", [False, True])
+    @pytest.mark.parametrize("allow_duplicates", ["absent", None, False, True])
```

Comment:
this is too confusing. don't use absent. instead you can set the option_context for both True & False (it could be a separate test is easier). having a single giant parameterized tests is sometimes not ok

Reply:
Not sure what you mean by "option_context", but I have put the "absent" option into a separate test and simplified this one.